src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,48 @@
+package mate.jdbc.model;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver(String name, String licenseNumber) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {
+        this.id = id;
+        this.name = name;
+        this.licenseNumber = licenseNumber;
```

Comment:
```suggestion
        this.id = id;
        this(name, licenseNumber);
```

It's better to use one constructor into another